travis.sh

```diff
@@ -141,8 +142,10 @@ my_travis_wait 60 catkin build --no-status --summarize || exit 1
 # Source the new built workspace
 travis_run source install/setup.bash;
 
-# Only run tests on the current repo's packages
-TEST_PKGS=$(catkin_topological_order $CI_SOURCE_PATH --only-names)
+# Choose which packages to run tests on
+echo "Test blacklist: $TEST_BLACKLIST"
+TEST_PKGS=$(catkin_topological_order src --only-names | grep -Fvxf <(echo "$TEST_BLACKLIST" | tr ' ;,' '\n'))
```

Comment:
I think this feature is still good for temp disabling tests in a world where we have a bunch of packages in the same moveit repo and we can't fix a test at the moment, so i think we should keep it

could you fix the feature that makes it only build tests for the target folder though? or i guess that is already working we just had a strange usecase here